test/unit/typedQueryBuilderTests.ts

```diff
@@ -1325,9 +1325,9 @@ describe('TypedKnexQueryBuilder', () => {
 
     it('should handle where exists on same table as main table', (done) => {
         const typedKnex = new TypedKnex(knex({ client: 'postgresql' }));
-        const query = typedKnex.query(User).whereExists(User, (subQuery => {
+        const query = typedKnex.query(User).whereExists(User, (subQuery) => {
```

Comment:
All of the other subquery callback functions use this syntax so I changed this to match that (I personally prefer the no parentheses syntax but that's just me :) )

Reply:
Prettier thinks this is the way ...

Reply:
Guess we can't say no to something pretty ☺️